custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -355,7 +373,12 @@ def _state_aggregation_query(self, state_id):
             ("birth_weight", "child_health.birth_weight"),
             ("child_person_case_id", "child_health.mother_id"),
             ("delivery_nature", "del_form.delivery_nature"),
-            ("term_days", "(del_form.add::DATE - del_form.edd::DATE) + 280")
+            ("term_days", "(del_form.add::DATE - del_form.edd::DATE) + 280"),
+            ("valid_in_month_end", "CASE WHEN {} THEN 1 ELSE 0 END".format(valid_in_month_end)),
+            ("not_migrated_month_end", "CASE WHEN {} THEN 1 ELSE 0 END".format(not_migrated_month_end)),
+            ("alive_in_month_end", "CASE WHEN {} THEN 1 ELSE 0 END".format(alive_in_month_end)),
+            ("status_duplicate", "CASE WHEN person_cases.reason_closure='dupe_reg' THEN 1 ELSE 0 END"),
```

Comment:
Spec also mentions about `incorrect_reg` value. Do we need to use that as well here?

Reply:
good to also add a check of if closed_status_daily=1 and person_cases.reason_closure='dupe_reg' then duplicate else 0 Because we should not set this status for the child who is closed next month

Reply:
done